README.md

```diff
@@ -14,7 +14,7 @@ On Debian-based Linux distributions you can do it as follows:
 ```sh
 sudo apt-get install libcurl4-openssl-dev libjson-c-dev cmake binutils make
 ```
-On Mac OSX, first insall macports from [MacPorts](https://www.macports.org/install.php) and in Terminal
```

Comment:
Good catch, but Macports installation is a separate step than the command that follows. So, you don't want to remove and here.

Reply:
@ozbek Ok, thank! Returned 'and' to its place 👍